src/networks/mainnet.ts

```diff
@@ -472,6 +472,13 @@ export const mainnetContracts: NetworkContractAddresses = {
             ]
         }
     ],
+    merkleDistributors: [
```

Comment:
Most likely, there will only be one distributor for PENDLE, can have it under MISC

Reply:
Done as requested. We now assume only PENDLE tokens will be distributed.